File: download/downdemo/SendDemo.js
/**
 * 启瑞云短信发送测试demo
 * @author qiruiyun
 */
 
var request = require('request');
var querystring = require('querystring');

class SendDemo{
	
    constructor(apiKey, apiSecret){
		this.baseUrl = 'http://api.qirui.com:7891/mt';
        this.apiKey = apiKey;
        this.apiSecret = apiSecret;
    }
	
	getRequestUrl(mobile, message) {
		let data = {dc:'15', un:this.apiKey, pw:this.apiSecret, da:mobile, sm:message, tf:'3', rf:'2', rd:'1'};
		return this.baseUrl + '?' + querystring.stringify(data);
	}
	
	sendMessage(mobile, message) {
		let smsUrl = this.getRequestUrl(mobile, message);
		//console.log(smsUrl);
		request(smsUrl, function (error, response, body) {
			//打印返回结果
			console.log(body);
		});
	}
	
}

//APIKey(用户名)
var apiKey    = 'xxxxxxxxxx';
//APISecret(密码)
var apiSecret = 'xxxxxxxxxxxxxxxxxxx';
//接受短信的手机号
var mobile    = '15100000000';
//短信内容(【签名】+短信内容)，系统提供的测试签名和内容，如需要发送自己的短信内容请在启瑞云平台申请签名和模板
var message   = '【启瑞云】您的验证码是:3288';


var send = new SendDemo(apiKey, apiSecret);
send.sendMessage(mobile, message);

File: download/downdemo/SendDemo.java
package com.qirui.senddemo;

import java.net.URLEncoder;
import org.apache.http.HttpEntity;
import org.apache.http.client.config.RequestConfig;
import org.apache.http.client.methods.CloseableHttpResponse;
import org.apache.http.client.methods.HttpGet;
import org.apache.http.impl.client.CloseableHttpClient;
import org.apache.http.impl.client.HttpClients;
import org.apache.http.util.EntityUtils;

public class SendDemo 
{
    public static void main(String[] args) throws Exception 
	{
       
	    String url       = "http://api.qirui.com:7891/mt";

	    //apiKey和APISecret
        String apiKey    = "xxxxxxxxxx";
        String apiSecret = "xxxxxxxxxxxxxxxxxxx";
        
        //接受手机号
        String mobile    = "18500000000";
		//短信内容(【签名】+短信内容)，系统提供的测试签名和内容，如需要发送自己的短信内容请在启瑞云平台申请签名和模板
        String message   = "【启瑞云】您的验证码是:5277";
        
        StringBuilder sb = new StringBuilder(2000);
        sb.append(url);
        sb.append("?dc=15");
        sb.append("&sm=").append(URLEncoder.encode(message, "utf8"));
        sb.append("&da=").append(mobile);
        sb.append("&un=").append(apiKey);
        sb.append("&pw=").append(apiSecret); 
        sb.append("&tf=3&rd=1&rf=2");   //短信内容编码为 urlencode+utf8
        
        String request = sb.toString();
        //System.out.println(request);
        
		
		CloseableHttpClient client = HttpClients.createDefault();

    	HttpGet httpGet = new HttpGet(request);
		
		CloseableHttpResponse response = client.execute(httpGet);

		String respStr = null;
    
  		HttpEntity entity = response.getEntity();
        if(entity != null) {
              respStr = EntityUtils.toString(entity, "UTF-8");
        }
        System.out.println(respStr);
			 
    }
	
}